history/package/marathon.json.mustache

```diff
@@ -2,27 +2,59 @@
     "id": "{{service.name}}",
     "cpus": {{service.cpus}},
     "mem": {{service.mem}},
```

Comment:
is the `user` configurable on the history server?

Reply:
I'll make it configurable and change the default to `nobody`.